concrete/src/System/Info.php

```diff
@@ -466,12 +462,23 @@ public function getDbVersion()
     {
         return $this->dbVersion;
     }
-    
+
     /**
      * @return string
      */
     public function getDbCharset()
     {
+        if ($this->dbCharset === null) {
+            $this->dbCharset = '';
+            if ($this->installed) {
+                try {
+                    $resolver = $this->app->make(Resolver::class);
+                    $db = $this->app->make('database')->connection();
+                    [$this->dbCharset, $dbCollationIgnored] = $resolver->resolveCharacterSetAndCollation($db);
+                } catch (\Exception $x) {
```

Comment:
@aembler In order to catch everyting, I'd use `\Throwable` instead of `\Exception`